### ISSUE TITLE
grpc_subscription_autoreconnect_tasks errors: failed to create filter: Broadcast `any` is not allowed

### ISSUE THREAD

**Issue by McSim85** [blockworks-foundation/lite-rpc#398]
Hey team

Just wanted to post this issue more like an FAQ, if anyone else meets this.
`lite-rpc` can report these errors:
```sh
WARN geyser_grpc_connector::grpc_subscription_autoreconnect_tasks: error on grpc_addr http://X.Y.Z.W:10000/ - retrying: Status { code: InvalidArgument, message: "failed to create filter: Broadcast `any` is not allowed, at least one filter required", source: None }
```
The error comes from this part of plugin:
https://github.com/rpcpool/yellowstone-grpc/blob/b9d629c2fcd570e24040bf7784757981ca9be86e/yellowstone-grpc-geyser/src/config.rs#L216

This happens because `"account_include_any"` parameter (`grpc -> filters -> blocks -> account_include_any`) in the yellostone GRPC `config.json` has to be `true`.

rel: https://github.com/blockworks-foundation/lite-rpc/issues/307

**Comment by pelaricoaus** [comment]
yeah but that yellowstone-grpc-geyser isnt apart of the dependencies for lite-rpc?? only yellowstone-grpc-proto & yellowstone-grpc-client... so how to fix this?
